### ISSUE TITLE
Logger not available in Objective-C project

### ISSUE THREAD

**Issue by joaomvfsantos** [piwik/piwik-sdk-ios#185]
I'm using this SDK on an objective c project, however i cannot access the logger property of the PiwikTracker class, or any Logger class for that matter.

```objc
#import "PiwikTracker-Swift.h"
// (...)
[PiwikTracker configureSharedInstanceWithSiteID:siteId baseURL:piwikUrl];
[PiwikTracker shared].logger = (...) // compile error: Property 'logger' not found on object of type 'PiwikTracker *'    
[PiwikTracker shared].dispatchInterval = 10;
```

Am I missing something here? Thanks in advance.

**Comment by brototyp** [comment]
Hi @joaomvfsantos, this can very well be true. I will have a look into it.

**Comment by brototyp** [comment]
Hi @joaomvfsantos, I just checked this out. Please try out this branch `feature/185-logger-in-objc` and use something like `[PiwikTracker shared].logger = [[DefaultLogger alloc] initWithMinLevel:LogLevelVerbose];` to define your log level. 

Please get back to me if you tried it.

**Comment by joaomvfsantos** [comment]
Hello @brototyp I'll try to have a look at this in the next few days. I'll keep in touch.

**Comment by joaomvfsantos** [comment]
@brototyp This seems to be working. Thanks

**Comment by brototyp** [comment]
@joaomvfsantos Great. I will merge it soon. It will be part of the next release.

**Comment by brototyp** [comment]
This just got merged and will be part of the next release

**Comment by brototyp** [comment]
Just released [version 4.4.0](https://github.com/piwik/piwik-sdk-ios/releases/tag/v4.4.0) containing this change.
